artemis-console/pom.xml

```diff
@@ -21,40 +21,36 @@
 
   <parent>
     <groupId>org.apache.activemq</groupId>
-    <artifactId>artemis-hawtio-pom</artifactId>
+    <artifactId>artemis-pom</artifactId>
     <version>2.36.0</version>
+    <relativePath>../artemis-pom/pom.xml</relativePath>
   </parent>
 
   <artifactId>artemis-console</artifactId>
   <name>ActiveMQ Artemis Console</name>
 
   <packaging>war</packaging>
 
-  <properties>
-    <activemq.basedir>${project.basedir}/../..</activemq.basedir>
-  </properties>
-
   <dependencies>
     <dependency>
-      <groupId>org.eclipse.jetty.toolchain</groupId>
-      <artifactId>jetty-servlet-api</artifactId>
+      <groupId>jakarta.servlet</groupId>
+      <artifactId>jakarta.servlet-api</artifactId>
```

Comment:
This used the jetty dep before as we consolidated on it to avoid having effectively duplicate bits in the distribution and the build. This will cause that again. Not sure it needs changing other than adjusting to the newer jetty releases dep (jetty-jakarta-servlet-api).